src/gov/nist/core/LexerCore.java

```diff
@@ -44,6 +44,7 @@
     public static final int START = 2048;
     public static final int END = START + 2048;
     // IMPORTANT -- This should be < END
+    public static final int IPV6 = END - 4;
```

Comment:
public static final int IPV6 = END + 4; 
we also have the L_SQUARE_BRACKET value and maybe can apply it to our code, but I think IPV6 will be more easy to understand the code when we are trying to parse IPV6 address.

I have checked in the code. Could you take a look a bit?